js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -32,11 +32,11 @@ var WIDGET_CLASS = "dx-treeview",
     ITEM_DATA_KEY = "dx-treeview-item-data",
     IS_LEAF = "dx-treeview-node-is-leaf",
     EXPAND_EVENT_NAMESPACE = "dxTreeView_expand",
-    TOGGLE_ITEM_VISIBILITY_CLASS = "dx-treeview-toggle-item-visibility",
```

Comment:
We should discuss about the possible breaking change with @jsdmitry and support team